src/v3/webpack.common.config.ts

```diff
@@ -48,37 +73,24 @@ const baseConfig: Partial<Configuration> = {
             '/node_modules/p-cancelable',
           ].some(filePathContains);
           const shallBeExcluded = [
+            // /src/ will be handled in next rule
+            resolve(__dirname, '..'),
             '/node_modules/',
             'packages/@okta/qtip2',
           ].some(filePathContains);
 
           return shallBeExcluded && !npmRequiresTransform;
         },
         loader: 'babel-loader',
-        options: {
-          sourceType: 'unambiguous',
-          presets: [
-            [
-              '@babel/preset-env',
-              {
-                // TODO: resolve issue with authjs esm bundle, then switch to "usage" to reduce polyfill size
-                useBuiltIns: 'entry',
-                corejs: '3.9',
-              },
-            ],
-            '@babel/preset-react',
-            '@babel/preset-typescript',
-          ],
-          plugins: [
-            [
-              '@babel/plugin-transform-react-jsx',
-              {
-                runtime: 'automatic',
-                importSource: 'preact',
-              },
-            ],
-          ],
-        },
+        options: babelOptions,
+      },
+      {
+        test: /\.[jt]sx?$/,
```

Comment:
Can you add a comment mentioning we need to separate these because of the dev config? Just so future developers don't see this and think they can simplify by combining these.